image_proc/src/rectify.cpp

```diff
@@ -51,9 +51,16 @@ namespace image_proc
 RectifyNode::RectifyNode(const rclcpp::NodeOptions & options)
 : rclcpp::Node("RectifyNode", options)
 {
-  auto qos_profile = getTopicQosProfile(this, "image");
+  // TransportHints does not actually declare the parameter
+  this->declare_parameter<std::string>("image_transport", "raw");
```

Comment:
include header `<string>`

Reply:
string is already in the hpp for this file - hence why the linter isn't asking for it to be added to the cpp file